tests/Behat/Page/Admin/OrderCreatePage.php

```diff
@@ -112,4 +120,23 @@ private function getItemWithProductSelected(int $productId): NodeElement
 
         throw new \Exception(sprintf('There is no item with product with id "%d" selected', $productId));
     }
+
+    private function clickOnTabAndWait(string $tabName): void
+    {
+        $tab = $this->getDocument()->find('css', sprintf('.title:contains("%s")', $tabName));
+
+        if ($tab->hasClass('active')) {
+            return;
+        }
+
+        $tab->click();
+
+        $this->getDocument()->waitFor(10, function () use ($tabName) {
```

Comment:
Since builds on travis are fixed, can you try reducing this amount of time?

Reply:
Fixed